generator/bindata.go

```diff
@@ -1,66 +1,66 @@
 // Code generated by go-bindata. DO NOT EDIT.
 // sources:
-// templates/cli/cli.gotmpl (8.129kB)
```

Comment:
Are you using "github.com/kevinburke/go-bindata/go-bindata" ? 
The files are off.

Reply:
@casualjim Do we have a bindata gen check in CI to ensure that data are correctly generated?

Reply:
I used github.com/shuLhan/go-bindata/. I will regenerate with github.com/kevinburke/go-bindata/go-bindata and will let you know.

Reply:
Done. Now the files look more similar to what's in master.